src/coreclr/src/tools/aot/crossgen2/crossgen2.csproj

```diff
@@ -42,7 +42,7 @@
 
   <PropertyGroup>
     <CrossHostArch></CrossHostArch>
-    <CrossHostArch Condition="'$(TargetArchitecture)' == 'arm64' and '$(BuildArchitecture)' == 'x64'">x64</CrossHostArch>
+    <CrossHostArch Condition="'$(TargetArchitecture)' == 'arm64' and '$(CLR_CMAKE_HOST_ARCH)' == 'x64'">x64</CrossHostArch>
```

Comment:
Where do we set `CLR_CMAKE_HOST_ARCH` so that it shows up in msbuild? I thought `CLR_CMAKE_HOST_ARCH` is for cmake only.

Reply:
I also believe this is CMake only.